prowler/lib/utils/utils.py

```diff
@@ -17,9 +17,14 @@ def open_file(input_file: str, mode: str = "r") -> TextIOWrapper:
     try:
         f = open(input_file, mode)
     except Exception as e:
-        logger.critical(
-            f"{input_file}: {e.__class__.__name__}[{e.__traceback__.tb_lineno}]"
-        )
+        if e.__class__.__name__ == "OSError":
```

Comment:
Why don't you do `except OSError as err:`?

I think is clearer and then you can handle other exceptions

Reply:
Sure!